src/components/panel/panel.js

```diff
@@ -1001,9 +1009,7 @@ MdPanelService.prototype._openCountExceedsMaxOpen = function(groupName) {
  * @private
  */
 MdPanelService.prototype._closeFirstOpenedPanel = function(groupName) {
-  if (groupName) {
-    this._groups[groupName].openPanels[0].close();
-  }
+  this._groups[groupName].openPanels[0].close();
```

Comment:
This function is actually only fired from the `_openCountExceedsMaxOpen` method above it. That method checks if the current number of open panels within the group exceeds the maximum limit and if so, the `_closeFirstOpenedPanel` method is called to close the first opened panel.